frontend/index.jsx

```diff
@@ -143,7 +145,7 @@ class App extends React.Component {
     console.log('in update feed: ', images);
     const newImageFeed = this.state.imageFeed;
     newImageFeed.push(images);
-    newImageFeed.sort((imageA, imageB) => { return imageA.created < imageB.created});
+    newImageFeed.sort((imageA, imageB) => { return imageA.created > imageB.created});
```

Comment:
The sorting direction was ok, please revert this.

Reply:
Oh, I think I see, why you did that. I said, that "the user-feed needs to be sorted in reverse". What I meant was the feed of the logged in user, not the subscriber feed.

Reply:
But you don't need to do that. We can do that in another step.